r/R/dataset-scan.R

```diff
@@ -174,8 +174,6 @@ ScanTask <- R6Class("ScanTask",
 #' a `data.frame` for further aggregation, even if you couldn't fit the whole
 #' `Dataset` result in memory.
 #'
-#' This is experimental and not recommended for production use.
```

Comment:
We might want to still keep the experimental label here — it's working now, but as you mention, we might refactor it / have it possibly have different behavior in the future.

Reply:
I think the `do_exec_plan` refactor wouldn't affect behavior of this. But not sure about the "wrap in `arrow_dplyr_query`" one.

If we keep it experimental, we should either remove the vignette below or mark that as experimental as well.

Reply:
I think the vignette is helpful and useful. I would lean (slightly) towards marking this as experimental in both places, but only pretty weakly, happy to go with it if you would prefer to take the experimental marks off.

Reply:
Okay, I've marked both as experimental.